### ISSUE TITLE
Bomb is decreasing points now

### ISSUE THREAD

**Issue by syedsohail00** [devbutlazy/Blum-AutoClicker#23]
Whenever it clicks on bomb, the points are getting decreased, how do make it stop clicking on bombs?

**Comment by Mortal792** [comment]
yeah the bomb previously gave you 20 points, but seems like they reset it to the default -100 points again

**Comment by IsaacWorldTaker** [comment]
This seems to be intentional by the devs lol! because bombs have so much in common with the other shapes in terms of color, especially since there are two types of bombs. Finding a distinct feature in their color that sets them aside from the other ornaments is very difficult now (or maybe impossible!)

**Comment by Alimrd200001** [comment]
f the devs for that, I should have used all my tickets last night before this happened

**Comment by 7urkm3n** [comment]
Interesting, telegram admins think they are GOD in the channel and cant commit minor changes now ?

**Comment by Abbas121133** [comment]
can u update ?
